spec/support/helpers/landing_pages.rb

```diff
@@ -21,6 +21,6 @@ def get_landing_pages(locale_obj = Capybara.app.default_locale_obj)
       model
     end
 
-    return collection
+    collection
```

Comment:
But then how will I know it returns it?  You'll be making thousands of commits like these, good luck.

Reply:
Ruby always returns the last result. And I'm not going through all source files and change those things. Only when I touch the file anyhow.